### PR TITLE
Fixing Typos

### PR DESCRIPTION
**Pull request by nsp4746**
Fixing some typos, i.e. consistency of spaces. 

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by t8m** [comment]
Please drop the merge commit, these are not allowed. Also the commits can be squashed into one commit and the `CLA: trivial` annotation must be on a separate line in the git commit message body.

You can use `git rebase -i; git push --force` to edit your commits.

File: CHANGES.md [hunk 1]
```diff
@@ -16283,11 +16283,11 @@ s-cbc           3624.96k     5258.21k     5530.91k     5624.30k     5628.26k
    2. Fix logical glitch in is_MemCheck_on() aka CRYPTO_is_mem_check_on().
 
    3. Count how many times MemCheck_off() has been called so that
-      nested use can be treated correctly.  This also avoids
+      nested use can be treated correctly. This also avoids
       inband-signalling in the previous code (which relied on the
       assumption that thread ID 0 is impossible).
 
-   *Bodo Moeller*
+   *Bodo Moeller*
```
Comment (bbbrumley) [inline CHANGES.md:16290]: @nsp4746 looks like an accidental space insertion here

**Comment by InfoHunter** [comment]
You may also add `[skip ci]` in the commit message to skip the CI since this contains doc changes only. @nsp4746

**Comment by paulidale** [comment]
The reason being that the CIs sanity check the docs.

**Pull request closed without merging**